README.md

```diff
@@ -19,19 +19,20 @@ Maven plugin for the automated building of Axon Ivy Projects.
 
 #### Release
 
-1. Consider updating the default engine version in the [AbstractEngineMojo](src/main/java/ch/ivyteam/ivy/maven/AbstractEngineMojo.java#L40) 
```

Comment:
this "consideration" is already addressed in 'prepare' so we should not duplicate it.